### ISSUE TITLE
notify when new contacts have registered for TextSecure

### ISSUE THREAD

**Issue by mcginty** [WhisperSystems/TextSecure#2553]
A small positive reinforcement when you can communicate with new friends privately :).

**Comment by donjoe0** [comment]
I suppose there's no point in suggesting that those contacts should have control over whether you get notified about their registration or not - my similar suggestion for message reception notifications was closed with no explanation.

**Comment by d3k4r** [comment]
I personally would not like to see this implemented. I saw it on one other app, Telegram I think, and it was really annoying. Could have been just a bad implementation of it though..

**Comment by donjoe0** [comment]
Yep, that's exactly what turned me off of Telegram and made me uninstall it right away: someone in my phone contacts that I wasn't really that close to - and didn't want to be - mentioned to me that they had seen me install Telegram, with the exact day and hour when that happened. I'm not used to people knowing what and when I'm installing on my phone without me telling them and I don't think I want to get used to that much transparency, especially with people I'm not close to (even though I have their phone numbers and they have mine).

It looks like mobile developers haven't yet woken up to the fact that a phone contacts list is very much not the same thing as a social media "friends" list - people can exchange phone numbers for entirely business-related/institutional reasons, with no desire for any personal relationship whatsoever. Mobile apps that access the phone contacts list should adapt their behaviour to this reality and offer options to treat different contacts differently in terms relevant to "social proximity" or something like that.

**Comment by nwdev** [comment]
I agree. If implemented, please give us the option to disable it.  @donjoe0 raises a few valid points..

**Comment by Ste4thOverride** [comment]
I believe those are two different issues. I want to know when others install the app but I do not want others to know when I install the app. So one setting with two options should do it.
